crates/red_knot_test/src/parser.rs

```diff
@@ -133,8 +133,9 @@ struct EmbeddedFileId;
 
 /// A single file embedded in a [`Section`] as a fenced code block.
 ///
-/// Currently must be a Python file (`py` language) or type stub (`pyi`). In the future we plan
-/// support other kinds of files as well (TOML configuration, typeshed VERSIONS, `pth` files...).
+/// Currently must be a Python file (`py` language), a type stub (`pyi`) or a `VERSIONS` file.
```

Comment:
it might again be good to link to https://github.com/python/typeshed/blob/c546278aae47de0b2b664973da4edb613400f6ce/stdlib/VERSIONS#L1-L18 here, as many readers of this might not be familiar with the `VERSIONS` file